php/php.editor/test/unit/data/goldenfiles/org/netbeans/modules/php/editor/lexer/PHP53FeaturesTest/lexer/nowdoc_012.pass

```diff
@@ -11,22 +11,19 @@ token #9 WHITESPACE
 token #10 PHP_OPERATOR =
 token #11 WHITESPACE  
 token #12 PHP_NOWDOC_TAG_START <<<'ENDOFNOWDOC'\n
-token #13 PHP_CONSTANT_ENCAPSED_STRING This is a nowdoc test.\nNOTREALLYEND;
-token #14 PHP_CONSTANT_ENCAPSED_STRING \nAnother line\nNOTENDEITHER;
-token #15 PHP_CONSTANT_ENCAPSED_STRING \nENDOFNOWDOCWILLBESOON
-token #16 PHP_CONSTANT_ENCAPSED_STRING \nNow let's finish it\n
-token #17 PHP_NOWDOC_TAG_END ENDOFNOWDOC
-token #18 PHP_SEMICOLON ;
-token #19 WHITESPACE \n
-token #20 PHP_PRINT print
-token #21 WHITESPACE  
-token #22 PHP_CONSTANT_ENCAPSED_STRING "
-token #23 PHP_CURLY_OPEN {
-token #24 PHP_VARIABLE $x
-token #25 PHP_CURLY_CLOSE }
-token #26 PHP_ENCAPSED_AND_WHITESPACE \n
-token #27 PHP_CONSTANT_ENCAPSED_STRING "
-token #28 PHP_SEMICOLON ;
-token #29 WHITESPACE \n\n
-token #30 PHP_CLOSETAG ?>
-token #31 T_INLINE_HTML \n--EXPECT--\nThis is a nowdoc test.\nNOTREALLYEND;\nAnother line\nNOTENDEITHER;\nENDOFNOWDOCWILLBESOON\nNow let's finish it\n
+token #13 PHP_CONSTANT_ENCAPSED_STRING This is a nowdoc test.\nNOTREALLYEND;\nAnother line\nNOTENDEITHER;\nENDOFNOWDOCWILLBESOON\nNow let's finish it\n
+token #14 PHP_NOWDOC_TAG_END ENDOFNOWDOC
```

Comment:
In the case of nowdoc, I suppose that we should treat the body as one token because it does not include another token.